src/main/java/com/dj/core/model/graph/ConnectedNeuron.java

```diff
@@ -115,7 +115,7 @@ public void forwardSignalReceived(final Neuron from, final Double value) {
             }
 
             // Step #2
-            double signalToSend
+            final double signalToSend
```

Comment:
@b0noI I am not very good at Java10, but I think we can replace `final double` with `final var`.

Reply:
done